client-default/src/test/scala/com.gu.contentapi.client/GuardianContentClientBackoffTest.scala

```diff
@@ -69,24 +69,17 @@ class GuardianContentClientBackoffTest extends FlatSpec with Matchers with Scala
 
   private val TestItemPath = "commentisfree/2012/aug/01/cyclists-like-pedestrians-must-get-angry"
 
-  implicit val executor: ScheduledExecutor = ScheduledExecutor()
+  implicit val executor0: ScheduledExecutor = ScheduledExecutor()
 
-  "Client interface" should "establish the backoff strategy" in {
```

Comment:
This test is not testing any behaviour

Reply:
It was testing that the expected values were written into the strategy

Reply:
True, but we are mostly interested in testing the behaviour, i.e. effect of those values rather the values themselves.